src/components/BookList.js

```diff
@@ -1,24 +1,43 @@
 import { useSelector, useDispatch } from 'react-redux';
-import { delBook } from '../redux/books/bookSlice';
+import { useEffect } from 'react';
+import { delBook, getBooks, deleteBook } from '../redux/books/bookSlice';
 
 const BookList = () => {
-  const books = useSelector((state) => state.book);
+  const { books, isLoading } = useSelector((state) => state.book);
   const dispatch = useDispatch();
 
-  const deleteBook = (bookId) => {
-    dispatch(delBook(bookId));
-  };
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
+
+  if (isLoading) {
+    return (
+      <div className="loading">
+        <p>Loading...</p>
+      </div>
+    );
+  }
 
   return (
     <ul>
       {books.map((book) => (
-        <li key={book.id} className="book_list">
+        <li key={book.item_id} className="book_list">
           {book.title}
           {' '}
           -
           {' '}
           {book.author}
-          <button type="button" className="del_btn" onClick={() => deleteBook(book.id)}>Delete</button>
+          <button
+            type="button"
+            className="del_btn"
+            onClick={() => {
+              dispatch(delBook(book.item_id));
+              dispatch(deleteBook(book.item_id));
+            }}
+          >
+            Delete
+
+          </button>
         </li>
       ))}
     </ul>
```

Comment:
- [ Optional ] Good job in connecting the Bookstore App to the API, however, I've noticed that your styling doesn't follow the design provided on [Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). Since this is not a requirement for this milestone I'll mark it as an optional requirement for now. 
Please consider implementing this change since it will be a _core requirement_ in the next milestone of this project.

| Zeplin Design | Your Design |
|---------------------|------------------|
|![Screenshot from 2023-05-26 21-48-46](https://github.com/davidolanrewaju/Bookstore/assets/16392046/50d0ec4a-c678-4233-a7c6-87cf2dee6c25) | ![Screenshot from 2023-05-26 21-47-53](https://github.com/davidolanrewaju/Bookstore/assets/16392046/14aedfd6-9ffe-4791-af37-3b7aa97cfadb)|